src/dialogs/transcodedialog.ui

```diff
@@ -144,45 +144,44 @@
     <widget class="QWidget" name="advancedWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
-       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">

```

Comment:
This is the mess I get when I use QT Designer to edit the ui rather than hand-editing the XML. All I did was add one label and one QComboBox. But the tools seems to move a bunch of things around.

Reply:
Sometimes you need to break a layout and recreate it. In that case, you need to check the custom properties set on the layout beforehand in order to restore them. This is what was lost:
![image](https://user-images.githubusercontent.com/1146683/205182863-03bd28be-4782-4739-9e9a-4495a6b01c6a.png)

And yes, after changes the XML might look scary in a diff viewer.

Reply:
I cleaned this up and also right-aligned the field labels per our UI conventions.